src/add-label.js

```diff
@@ -0,0 +1,45 @@
+const core = require('@actions/core')
+
+/**
+ * createNewPullRequest
```

Comment:
Need to adjust the JSDoc function name.
```suggestion
 * addLabel
```

Reply:
Thanks for catching that up!